src/client/components/allBlocks/allBlocks.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import connect from 'react-redux/es/connect/connect';
+import {
+  setVisibility,
+} from '../../actions/blocksActions';
+import BlockOfSliders from '../blockOfSliders';
+import Button from '../button';
+import {
+  pauseIcon, playIcon, stopIcon, checkTickIcon,
+} from '../../assets/icons/icons';
+import Sound from '../../sounds/sound.mp3';
+import './alllBlocks.css';
+
+class AllBlocks extends Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      sound: new Pizzicato.Sound(Sound, this.runSound),
+      isPlaying: false,
+    };
+  }
+
+  runSound = () => {
+    const { blocksData } = this.props;
+    return blocksData.forEach(({ createEffect }) => {
+      const { sound } = this.state;
+      sound.addEffect(createEffect);
+    });
+  }
+  ;
+
+  play = () => {
+    this.setState({ isPlaying: true });
+    const { sound } = this.state;
+    sound.play();
+  };
+
+  pause = () => {
+    this.setState({ isPlaying: false });
+    const { sound } = this.state;
+    sound.pause();
+  };
+
+  stop = () => {
+    this.setState({ isPlaying: false });
+    const { sound } = this.state;
+    sound.stop();
+  };
+
+  setEffectsValue = (blockName, effectsName, value) => {
+    const { blocksData } = this.props;
+    blocksData.forEach(({ name, effects, createEffect }) => {
+      if (name === blockName) {
+        effects[effectsName] = value;
+        createEffect[effectsName] = effects[effectsName];
+      }
+    });
+  };
+
+  render() {
+    const { sound, isPlaying } = this.state;
+    const { setVisibility, blocksData } = this.props;
+
+    const PlayButton = (
+      <Button
+        className="PlayButton"
+        onClick={this.play}
+        icon={playIcon}
+        value="Play"
+      />
+    );
+
+    const PauseButton = (
+      <Button
+        className="PauseButton"
+        onClick={this.pause}
+        icon={pauseIcon}
+        value="Pause"
+      />
+    );
+
+    const StopButton = (
+      <Button
+        className="StopButton"
+        onClick={this.stop}
+        icon={stopIcon}
+        value="Stop"
+      />
+    );
+
+    return (
+      <div className="SlidersComponent__main--container">
+        <div className="AllSliders">
+          <div className="ListOfEffects">
+            {
+              blocksData.map(({ name, isVisible }, i) => (
+                <button
+                  className="Effect"
+                  key={i}
```

Comment:
It's not a good practise to set index as the keys in "map" expression. I would suggest to use "name" property instead.

Reply:
Got it. Fixed